com.zen.browser.metainfo.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
```

Comment:
You are missing multiple mandatory tags.

Validate the file https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#validation and add them.

The docs above has the info on required tags.